pybossa/util.py

```diff
@@ -955,18 +955,19 @@ def valid_or_no_s3_bucket(task_data):
 
 
 def can_update_user_info(current_user, user_to_update):
+    disable_fields = {'user_type': 'You must be an admin or subadmin to edit this.'}
+    hidden_fields = {'profile': 'You must be an admin or subadmin to view and edit this.'}
     # admin can update anyone
     if current_user.admin:
-        return True, None
+        return True, None, None
     # subadmin can update self and normal users
     if current_user.subadmin:
         return (current_user.id == user_to_update.id or
-            not (user_to_update.admin or user_to_update.subadmin)), None
-    # normal user can update self except for 'user_type' field
+            not (user_to_update.admin or user_to_update.subadmin)), None, None
+    # normal user cannot view/update 'profile' field, cannot update 'user_type' field
```

Comment:
by normal, i believe its regular or non-admin/non-subadmin user? if so, and you want to change comment, similar change applies to variable defined as normal_user

Reply:
Right, normal=don't have admin or subadmin

Reply:
should we term it as regular user under comments and variable name then?

Reply:
will update comment and where is the variable name you mentioned?

Reply:
https://github.com/bloomberg/pybossa/blob/697ac81b579f4625cb84f8a16ab2e23ab84c669c/test/test_util.py#L1136-L1160